Scripts/DCS-BIOS/lib/modules/aircraft_modules/M-2000C.lua

```diff
@@ -0,0 +1,896 @@
+module("M-2000C", package.seeall)
+
+local Functions = require("Functions")
+
+local Module = require("Module")
+
+--- @class M_2000C: Module
+local M_2000C = Module:new("M-2000C", 0x7200, { "M-2000C" })
+
+--v1.39 by Ergo,Matchstick,MisterKnife,WarLord,Espresso29470
+
+--remove Arg# Pilot 1000
+
+local function round(num, numDecimalPlaces, returnstring)
+	if returnstring == nil then
+		returnstring = false
+	end
+	if returnstring then
+		return string.format("%." .. (numDecimalPlaces or 0) .. "f", num)
+	else
+		return tonumber(string.format("%." .. (numDecimalPlaces or 0) .. "f", num))
+	end
+end
+
+--Get Displays Functions
+
+-- parse radios
+local vhfFrequency = ""
+local uhfFrequency = ""
+
+M_2000C:addExportHook(function()
+	local radios = M_2000C.parse_indication(7)
+
+	local vhf = coerce_nil_to_string(radios.text_COM_UHF1)
+	if vhf then
+		vhfFrequency = vhf:sub(1, 3) .. vhf:sub(5, 6)
+	end
+
+	local uhf = coerce_nil_to_string(radios.text_COM_UHF2)
+	if uhf then
+		uhfFrequency = uhf:sub(1, 3) .. uhf:sub(5, 6)
+	end
```

Comment:
`uhf` can't be nil since `coerce_nil_to_string` is called (and we should be using the one in `Functions`)
```suggestion
	local uhf = Functions.coerce_nil_to_string(radios.text_COM_UHF2)
	uhfFrequency = uhf:sub(1, 3) .. uhf:sub(5, 6)
```

Reply:
fixed